### ISSUE TITLE
A string value surrounded by double quotes, are inconsistent.

### ISSUE THREAD

**Issue by nmarshall23** [paradigmatic/Configrity#25]
Let me demonstrate. 

``` scala
import org.streum.configrity._
val config = Configuration()
val config2 = config.set("foo",""""bar bar"""")
config2.save("foo.conf")
val config3 = Configuration.load("foo.conf")

val res1 = config2[String]("foo")
val res2 = config3[String]("foo")
```

Results in `res1` equaling `"bar bar"`. However `res2` equals `bar bar`, that is a string that doesn't contain double quotes. What is happening is that while loading Configrity strips off the double quotes. 

I think that to be consistent Configrity should keep the double quotes.

**Comment by paradigmatic** [comment]
Yes, I should rewrite the basic formats to be more precise about quotation rules (see issue #24 for instance). The problem is that I won't have much time until november, and that I am not a parser/regex specialist. So unfortunately I won't be able to fix this problem next week without help...
